src/settings.js

```diff
@@ -197,38 +197,48 @@ function getCliArgs(options = CLIOPTIONS, inputArgs = process.argv) {
  * @return {object}         - An object with errors and a boolean check
  */
 function checkCliInput(cliArgs, options = CLIOPTIONS) {
+	const result = {
```

Comment:
Need to add the headline
```suggestion
	D.header('checkCliInput', { cliArgs, options });
	const result = {
```